.gitignore

```diff
@@ -1,3 +1,5 @@
 dist/
+.stack-work/
 *.o
 *.hi
+TAGS
```

Comment:
It's a common file generated by setups that provide a `go to definition` facility (in my case it's [`hasktags`](http://hackage.haskell.org/package/hasktags)). Tell me if you prefer that to be removed